api/src/main/java/io/grpc/LoadBalancerProvider.java

```diff
@@ -86,15 +86,16 @@ public final String toString() {
   }
 
   /**
-   * Uses identity equality.
+   * Equal provider instances must be of the same class, have the same policy name, priority and
```

Comment:
It's not appropriate to mention details that only applies to subclasses here (e.g., not all load balancers have lb configs).

Reply:
Here it's the provider not balancer that parses lb config. 

``` java
public ConfigOrError parseLoadBalancingPolicyConfig(Map<String, ?> rawLoadBalancingPolicyConfig)
```

The config could be empty or not relevant to the balancer at all.

Reply:
Well, I just realized that `parseLoadBalancingPolicyConfig(...)` is an instance method for `LoadBalancerProvider` (even for `PickFirstLoadBalancerProvider` it provides an implementation that returns a `NO_CONFIG`).

Well. I would leaning for not saying anything here. And rather, implementors should know what they are doing.

Reply:
I just noticed that `LoadBalancerProvider` has a much simpler super class `LoadBalancer.Factory`, and that is all I need now.